plugins/yakshave.rb

```diff
@@ -0,0 +1,11 @@
+class Yakshave < Linkbot::Plugin
+    def self.on_message(message, matches)
+      "https://i.imgur.com/t0XHtgJ.gif"
+    end
+
+    Linkbot::Plugin.register('yakshave', self,
+      {
+        :message => {:regex => /(yakshave|yak shave|shaving.*yak)/i, :handler => :on_message}
```

Comment:
This probably won't be as annoying as unconstrained "no" and "cook" plugins where ... but it could be. Open to making it a bang command instead.